Assets/Scripts/Bee/BeeTutorialFlying.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using VRTK;
+
+public class BeeTutorialFlying : BeeTutorialBaseClass
```

Comment:
inherit from MonoBehaviour instead of BeeTutorialBaseClass since we're not using the isDone thing anymore